packages/core/src/config.ts

```diff
@@ -247,6 +247,44 @@ export const composeAutoExternalConfig = (options: {
     : {};
 };
 
+export function composeMinifyConfig(
+  minify: NonNullable<RsbuildConfig['output']>['minify'],
+): RsbuildConfig {
+  if (minify !== undefined) {
+    return {
```

Comment:
Should we delete use's `output.minify` config before final config merge process. Or we can return `{}` here, and final merge process will ship user's minify config

Reply:
Nice catch, I returned `{}` here.